### DISCUSSION TITLE
Next build errors on Github build: Specified format "sourceRelative" does not match any available codec.

### DISCUSSION THREAD

**Discussion by khuezy** [vercel/next.js#19938] [Help]
Hi, my project builds fine locally but when I try to build (next build) on Github actions, it has this error:

```
(node:16) UnhandledPromiseRejectionWarning: Error: Specified format "sourceRelative" does not match any available codec.
    at process (/app/node_modules/adjust-sourcemap-loader/lib/process/index.js:68:15)
    at onSuccess (/app/node_modules/resolve-url-loader/index.js:184:24)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:16) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:16) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
I was wondering if anyone else has any pointers. TIA.

Edit: this only happens using the built in sass loader. If I use next-sass, it works.
Edit 2: after many hours of debugging, adding this to the package.json resolved the issue: `"adjust-sourcemap-loader": "^4.0.0",` . I'm still not sure why it's failing w/o this lib though. It looks like whatever lib is using `resolve-url-loader` is using `adjust-sourcemap-loader` v3 which has some issues.

**Comment by makinde** [comment]
I'm seeing this as well, though installing `adjust-sourcemap-loader` doesn't fix for me :(

**Comment by makinde** [comment]
BTW, deleting my `node_modules` directory, then doing `yarn upgrade next@latest` fixed this for me.

**Reply by khuezy** [reply]
Nice, glad you're able to fix the problem. Were you experiencing this locally? It may be caching my build on Github Actions.
